src/cluster.py

```diff
@@ -171,3 +254,41 @@ def _start_patroni(self) -> bool:
         """
         service_start(PATRONI_SERVICE)
         return service_running(PATRONI_SERVICE)
+
+    def switchover(self, candidate: str = None) -> None:
+        """Schedule a switchover to a given candidate member????."""
```

Comment:
remove ???

Reply:
Fixed in https://github.com/canonical/postgresql-operator/pull/10/commits/bd497b27badc78c4db1d0de1b8eb5e956f9769de. Thanks!